### PR TITLE
Feature/account

### PR DESCRIPTION
**Pull request by mariush2**
- [AB#448463](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/448463)

### CODE REVIEW

File: src/components/Navigation/TopBar/index.ts [hunk 1]
```diff
@@ -2,12 +2,12 @@ import ApplicationDrawer from './ApplicationDrawer/ApplicationDrawer';
 import { GuidelineSections } from './Guidelines/Guidelines';
 import { Notifications } from './Notifications/Notifications';
 import { Resources } from './Resources/Resources';
-import { Account } from './Account';
 import { Actions } from './Actions';
 import FieldSelector from './FieldSelector';
 import Guidelines from './Guidelines';
 import { ISettingsProps, Settings } from './Settings';
 import { TopBar as BaseTopBar } from './TopBar';
+import { Account } from 'src/components/Navigation/TopBar/Account/Account';
```
Comment (aslakihle) [inline src/components/Navigation/TopBar/index.ts:10]: image

File: src/components/Navigation/TopBar/Account/Account.test.tsx [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import { faker } from '@faker-js/faker';
 
-import { render, screen, userEvent } from '../../../tests/test-utils';
-import { Account, AccountProps } from './Account';
+import {
+  Account,
+  AccountProps,
+} from 'src/components/Navigation/TopBar/Account/Account';
```
Comment (aslakihle) [inline src/components/Navigation/TopBar/Account/Account.test.tsx:6]: Shorten import i think, to relative path c:

File: src/components/Navigation/TopBar/TopBar.stories.tsx [hunk 3]
```diff
@@ -1,24 +1,19 @@
...
 
 import SideBarProvider from '../../../providers/SideBarProvider';
 import Template from '../../Template/Template';
 import SideBar from '../SideBar';
 import { MenuItemType } from '../SideBar/MenuItem';
 import TopBar from '.';
+import {
+  FAKE_ACCOUNT,
+  FAKE_ACCOUNT_PHOTO,
+} from 'src/components/Navigation/TopBar/Account/stories/data';
```
Comment (aslakihle) [inline src/components/Navigation/TopBar/TopBar.stories.tsx:14]: Same with this import

File: src/components/Navigation/TopBar/Account/Account.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import { AccountInfo } from '@azure/msal-common';
+import { Meta, StoryFn } from '@storybook/react';
+
+import { FAKE_ACCOUNT, FAKE_ACCOUNT_PHOTO } from './stories/data';
+import { Account } from 'src/components/Navigation/TopBar/Account/Account';
```
Comment (aslakihle) [inline src/components/Navigation/TopBar/Account/Account.stories.tsx:5]: Relative path here aswell (Idk which comment will be first in the list 😅 )

**Review by aslakihle — CHANGES_REQUESTED** [review]

File: src/components/Navigation/TopBar/index.ts (hunk 1 above)
Comment (aslakihle) [reply src/components/Navigation/TopBar/index.ts:10]: (Edited comment, so changed the order XD)

File: src/components/Navigation/TopBar/ApplicationDrawer/ApplicationDrawer.tsx [hunk 5]
```diff
@@ -9,7 +9,7 @@ import { TopBarButton } from '../TopBar.styles';
 import TopBarMenu from '../TopBarMenu';
 import { AmplifyApplication } from 'src/api/models/Applications';
 import { PortalService } from 'src/api/services/PortalService';
-import { ApplicationIcon } from 'src/components/index';
+import ApplicationIcon from 'src/components/Icons/ApplicationIcon/ApplicationIcon';
```
[the remark this answers is not in the document]
Comment (mariush2) [reply src/components/Navigation/TopBar/ApplicationDrawer/ApplicationDrawer.tsx:12]: For some weird reason the storybook breaks when trying to import from src/components. Think it has something to do with creating circular dependencies in our components

File: src/components/Navigation/TopBar/index.ts (hunk 1 above)
Comment (mariush2) [reply src/components/Navigation/TopBar/index.ts:10]: Same with this one

**Review by aslakihle — APPROVED** [review]

**Pull request merged**